### DISCUSSION TITLE
[EddieCon] - Thoughts, feedback, learning!

### DISCUSSION THREAD

**Discussion by naomi-lgbt** [EddieHubCommunity/support#2220] [Conference Notes]
Hey friends!

What's something really neat that you learned from EddieCon 0.1? What was your favourite topic? What are you most excited to try building?

What did you like about EddieCon? What do you think we should improve for next time? What feedback do you have?

Share your thoughts and experiences with EddieCon here!

**Comment by arinjayg** [comment]
Great Event and salute to Greater thought of celebrating + Adding Values to the community :)

**Reply by eddiejaoude** [reply]
Thank you, it was a great team and community effort 💥

**Comment by JagjiwanChimkar** [comment]
Awesome Event and Nicely managed. Learn many new things. Waiting for **EddieCon 0.2**

**Reply by eddiejaoude** [reply]
Thank you! Yay EddieCon 0.2 will be even better 🥳

**Comment by Sonichigo** [comment]
Intro to GraphQL and getting started with DataStax were two top for me!! Amazingly organized and hats off to every speakers !!

**Reply by eddiejaoude** [reply]
Great to know! Thank you for sharing 👍 we will do more of both 🥳

**Comment by Priyanshu-official** [comment]
**Eddie** it's really Awesome You made great day for all 🔥🔥🔥 And Happy Birthday🥳🥳🥳

**Reply by eddiejaoude** [reply]
Thank you! It was a great birthday for me 🥳

**Comment by Panquesito7** [comment]
The slash commands part was actually the best part for me, IMO. I liked how @nhcarrigan explained it so well. Overall, the live stream was great! I wasn't able to see it at all (I was sleeping), though, but it was great how all the people were talking and having fun. Happy birthday, Eddie! 😄🥳🎉

**Reply by eddiejaoude** [reply]
Thank you! I learnt a lot also 🥳

**Comment by ahamedbasha-n** [comment]
The Event was insightful and got to meet and interact with lot of people. It was fun and interesting at the end where I was unfortunate in the spin wheel. Looking forward for such amazing events. Thank You for the opportunity 💻❤.

**Reply by eddiejaoude** [reply]
Thank you for joining! Next time for spin the wheel 🥳

**Comment by kaiwalyakoparkar** [comment]
Very well conducted, Enjoyed and learnt a lot :tada:. Few things which I think could be improved
- **Time/speaker slot management:** (The time was managed well but calculating the buffer time would help the future as well)
- **Breaks:** We can take 5 mins break after every speaker session. This will give cool-down time to the viewers and will provide them time to bring in some snacks to have in on-going conference :sweat_smile: 
- **Feedback collection:** 15 mins before closing the stream we can circulate a form to get feedback as I see less people in feedback then in the actual event
- **Giveaway management:** As happened yesterday we should keep a google form or website form which will activate 10 minutes before the giveaway and the link to the form can be shared in the chat to be filled. So that present people could be rewarded in more simpler manner
- **Fun Events:** Adding fun events in the conf. It was still lot of fun tbh but adding small 5-10 mins comedy or art sessions would help in balancing the technical and fun stuff (I personally like these sessions, so this point is totally optional)

That's all. I really enjoyed it and looking forward to participate and speak at EddieCon 0.2

**Reply by shreemaan-abhishek** [reply]
Wonderful!!

**Reply by eddiejaoude** [reply]
Thank you for the awesome feedback back!! 🥳

**Comment by eddiejaoude** [comment]
Amazing day everyone, it was awesome to spend my birthday with you all and geek out! 💥 

Great feedback from everyone, we will definitely do some of those next time! Some of my thoughts:
- spilt the day into 2 days, that way it is easier for us but also allows more time for the hackathon submissions too
- breaks between talks for people to network more
- give aways for people who are there at the time, better way to capture this info
- more swag give aways

note: I will keep updating with more ideas

**Comment by ashwinexe** [comment]
Most of my feedback thoughts align with everyone here, I have some **ideas** for next time 
(maybe we can hava another discussion just to pour our ideas for next Con)

- An event in EddieCon which makes it stand-out from other events (i,e special focus on **Collaborate First, Code Second**) The even can follow around this spirit (it may be a talk, workshop, live hands-on, follow-along etc)
